tools/integration_tests/setup/setup.go

```diff
@@ -209,14 +209,24 @@ func ExecuteTestForFlags(flags [][]string, m *testing.M) (successCode int) {
 	return
 }
 
-func RunTests(flags [][]string, m *testing.M) {
+func ParseSetUpFlags() {
 	flag.Parse()
+}
+
+func RunTests(flags [][]string, testBucketValue string, m *testing.M) {
+	ParseSetUpFlags()
 
 	if !*integrationTest {
 		log.Printf("Pass --integrationTest flag to run the tests.")
 		os.Exit(0)
 	}
 
+	// If we set the bucket internally, we will ignore the values that will pass through the flag.
```

Comment:
For reviewer: We are running all the tests in integration_tests in kokoro script. for readonly tests we will internally set testbucket and we want to run tests on that testbucket only, so we should ignore --mountedDir flag value.

Reply:
since the read only tests are still not added, can we do these setup related changes seperately